code/datums/components/larryknife.dm

```diff
@@ -0,0 +1,43 @@
+//Larry With Knives
```

Comment:
This entire file is intended with spaces instead of tabs 😩

Reply:
Fixed this. Not sure how or why this happened.